README.md

```diff
@@ -12,6 +12,18 @@ This module provides an isomorphic render method for React components in an Expr
 $ npm install --save react-express-middleware react react-dom
 ```
 
+`index.html`
+```html
+<html>
+	<head></head>
+	<body>
+		<div id="app"><%- content %></div>
+	</body>
+</html>
+```
+
+### Option: Pass a single component
```

Comment:
I would also write a small paragraph about what this means, like "There are two ways to use this middleware's render method..."